src/datetime.rs

```diff
@@ -434,16 +367,15 @@ impl DateTime {
     /// let dt2 = dt.with_timezone_offset(Some(-8 * 3600)).unwrap();
     /// assert_eq!(dt2.to_string(), "2022-01-01T12:13:14-08:00");
     /// ```
-    pub fn with_timezone_offset(&self, offset: Option<i32>) -> Result<Self, ParseError> {
-        if let Some(offset_val) = offset {
+    pub fn with_timezone_offset(&self, tz_offset: Option<i32>) -> Result<Self, ParseError> {
+        if let Some(offset_val) = tz_offset {
             if offset_val.abs() >= 24 * 3600 {
```

Comment:
surely this logic should exist only in `time.rs`?

Reply:
I have re-used the `fmt` of Date and Time instead.